worker/uniter/uniter_test.go

```diff
@@ -50,37 +48,36 @@ var _ = gc.Suite(&UniterSuite{})
 var errNotDir = syscall.ENOTDIR.Error()
 
 func (s *UniterSuite) SetUpSuite(c *gc.C) {
-	s.GitSuite.SetUpSuite(c)
 	s.JujuConnSuite.SetUpSuite(c)
 	s.dataDir = c.MkDir()
 	toolsDir := tools.ToolsDir(s.dataDir, "unit-u-0")
 	err := os.MkdirAll(toolsDir, 0755)
 	c.Assert(err, jc.ErrorIsNil)
 
-	coretesting.CopyOrBuildJujuBins(c, toolsDir, "jujud", "jujuc")
-
 	s.oldLcAll = os.Getenv("LC_ALL")
-	os.Setenv("LC_ALL", "en_US")
+	err = os.Setenv("LC_ALL", "en_US")
```

Comment:
Don't we have a PatchEnvironment? Hmm.. I thought we did, but I can't find it. I guess this is still fine.

Reply:
We have it and I changed the test to use it